tests/kernel/ib_tests.pm

```diff
@@ -64,7 +87,7 @@ sub ibtest_master {
     # wait until the two machines under test are ready setting up their local things
     assert_script_run('cd hpc-testing');
     barrier_wait('IBTEST_BEGIN');
-    assert_script_run("./ib-test.sh $master $slave", 3600);
+    assert_script_run("./ib-test.sh $args $master $slave", 3600);
```

Comment:
Perhaps then it would make sense to allow to change the timeout? 
After all one can schedule only one 'phase' now, it can be also in VMs... that can be fast. No need to wait that long.

Reply:
Oof course it returns earlier. The test just fails if it takes longer. However, I like the configurable timeout through variables, I added that.

Reply:
ah OK; I see MPI tests problematic from this stand-point, but sure it make sense. thx for reply.